lib/src/views/post/new_post_screen.dart

```diff
@@ -526,6 +526,11 @@ class _NewPostScreenState extends State<NewPostScreen> {
                                     // maxLines: 200,
                                     decoration: const InputDecoration(
                                       border: InputBorder.none,
```

Comment:
We can wrap the whole `NewPostScreen` with `Theme` widget and pass our `ThemeData `instance, that will take care of the whole screen